google/cloud/speech/v1/cloud_speech_gapic.yaml

```diff
@@ -74,7 +74,7 @@ interfaces:
     request_object_method: true
     retry_codes_name: non_idempotent
     retry_params_name: default
-    timeout_millis: 60000
+    timeout_millis: 190000
```

Comment:
Technically, LongRunningRecognize doesn't need to be this long since we're just waiting for the LRO Name to be returned, not the actual recognition.  30 seconds would be more than long enough for the worst possible cases.  This probably isn't a critical issue though, is there any harm to making this too long?